researcher_ids/RI_Claims_V1.md

```diff
@@ -279,19 +286,128 @@ interfaces, such as:
 -   May provide a rich set of internationalization/localization features for
     clients to consume.
 
-## Claim Objects
+## RI Claim Embedded Token
 
 Each [RI claim](#researcher-identity-claim-ri-claim) name maps to an array of
-claim JSON objects ([RI Claim Objects](#researcher-identity-claim-object-ri-claim-object))
-within a "ga4gh" root OIDC claim object (see [example](#example-ri-claims)).
+([RI Claim Embedded Tokens](#researcher-identity-claim-embedded-token-ri-claim-embedded-token))
+which are encoded as
+[Embedded Tokens](https://github.com/ga4gh/data-security/blob/master/AAI/AAIConnectProfile.md#term-embedded-token)
+within a "ga4gh" root OIDC claim object.
+
+Non-normative example of a set of RI Claim Embedded Tokens, encoded as Embedded
+Token strings, with a RI Claim named "ResearcherStatus":
+
+```
+{
+  "ga4gh": {
+    "ResearcherStatus": [
+      "eyJhbGciOiJI...aaa",
+      "eyJhbGciOiJI...bbb"
+    ]
+  }
+}
+```
 
-## Claim Object Fields
+An [example](#example-ri-claims) of encoding multiple RI Claims in a more
+reader-friendly form is also available.
 
-Fields within a RI Claim Object are:
+### RI Claim Embedded Token Requirements
 
-#### "**value**" [required]
+-   RI Claim Embedded Tokens MUST conform to the
+    [GA4GH AAI Specification Embedded Token format](https://github.com/ga4gh/data-security/blob/master/AAI/AAIConnectProfile.md#embedded-token-issued-by-broker)
+    as a base64 string.
 
--   A string that represents the type, process and version of the claim. The
+-   Claim Signatories, Identity Brokers, Claim Clearinghouses, and their
+    Clients MUST conform to the
+    [GA4GH AAI Specification](https://github.com/ga4gh/data-security/blob/master/AAI/AAIConnectProfile.md)
+    requirements for Embedded Tokens in their use of RI Claim Embedded Tokens.
+
+-   Validation, as outlined elsewhere in this specification and the
+    GA4GH AAI Specification, MUST be performed before RI Claim
+    Embedded Tokens are used for identity or authorization.
+
+-   The "jti" is REQUIRED.
+
+-   If the OIDC Access Token format is used (i.e. the "scope" claim
+    contains the "openid" scope) as per the GA4GH AAI Specification's
+    Embedded Token encoding options, then the /userinfo endpoint is unused
+    but the /introspect endpoint MUST be available for
+    [claim polling](https://github.com/ga4gh/data-security/blob/master/AAI/AAIConnectProfile.md#claim-polling).
+
```

Comment:
It is not a requirement for an RI Claim Embedded Token to contain an "RI Claim Object" in the ga4gh_rio JWT claim?

Reply:
Removed this stuff as it is now covered better by the AAI spec.